defaults/main.yml

```diff
@@ -4,3 +4,4 @@ awx_repo_dir: "~/awx"
 awx_version: devel
 awx_keep_updated: yes
 awx_run_install_playbook: yes
+awx_host_port: 3001
```

Comment:
Is 3001 the current default? If it's 80, I'd like to keep the default 80.

Reply:
@geerlingguy you are right, is should be kept 80 and whoever wants to overwrite it, will change the port in group_vars/all.yml
Are you going to implement that change ?